src/commands/cmd_hash.cc

```diff
@@ -429,6 +430,238 @@ class CommandHRandField : public Commander {
   bool no_parameters_ = true;
 };
 
+class CommandFiledExpireBase : public Commander {
```

Comment:
Small typo. `CommandFiledExpireBase` (Fi**le**d) => `CommandFieldExpireBase`.

Reply:
Good catch!

Reply:
@torwig thank for you review, I will fix it :)